ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -133,6 +135,10 @@ public class ReplicaThread implements Runnable {
   // This is used in the test cases
   private Map<DataNodeId, List<ExchangeMetadataResponse>> exchangeMetadataResponsesInEachCycle = null;
 
+  public static final String REPLICA_STATUS_FILE = "onPremReplicaCheckStatus";
+  public static final String RECOVERY_STATUS_FILE = "cloudReplicaRecoveryStatus";
+  protected final BackupCheckerFileManager fileManager;
```

Comment:
`ReplicaThread` is a generic class used by all the use cases. Why does it need to have `BackupCheckerFileManager`?

Reply:
Same question. We'd better not touch Replication code. We don't need log for replication thread.

Reply:
1/ Not touching the replication code, its not logging anything. Default filemgr is empty methods.
2/ I want to log replication status from VCR to server in the DR node and this is the easiest way to do that. I have attempted creating a RecoveryThread here https://github.com/linkedin/ambry/pull/2359
And it seemed like way too much just to log recovery status.
3/ There are only two use cases of ReplicaThread : One to replicate between servers, and one for Recovery from cloud. This code does not interfere with either and is controlled by configs that I will use on DR node.

Reply:
If you have an easier/shorter or quicker way to do it, lmk.